config/samsung-fortuna3g.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From G530HXXU2BOH7_G530HODD2BOH3_G530HXXU2BOH5_HOME.tar.md5 INS-G530HXXU2BOH7-20150909215514.zip
+
+OPTIONS=(-r vddio -r vdd)
+PANELS=(
+	[s6d78a0_gh9607501a_qhd]="samsung,s6d78a0-gh9607501a"
+)
```

Comment:
```suggestion
OPTIONS=(-r vddio -r vdd --backlight-fallback-dcs --dcs-no-get-brightness)
PANELS=(
	[hx8389c_gh9607501a_qhd]="samsung,hx8389c-gh9607501a"
	[s6d78a0_gh9607501a_qhd]="samsung,s6d78a0-gh9607501a"
)
```

With https://github.com/msm8916-mainline/linux-mdss-dsi-panel-driver-generator/commit/64ec5ff19fe5fc18c6e71c710684ffcfcc624d3a